hazelcast/src/test/java/com/hazelcast/test/TestHazelcastInstanceFactory.java

```diff
@@ -187,7 +186,7 @@ protected static Address createAddress(String host, int port) {
 
     private static Config init(Config config) {
         if (config == null) {
-            config = new XmlConfigBuilder().build();
+            config = new Config(); //new XmlConfigBuilder().build();
```

Comment:
I believe so. It's already fixed in my local repo. I'll push it together with other changes addressing review comments. 
Serkan is commenting directly on commits, so force push could remove(?) his comments from this PR.